src/HashidsInterface.php

```diff
@@ -15,11 +15,14 @@ interface HashidsInterface
 {
     /**
      * Encode parameters to generate a hash.
-     * @param int|string|array<int, int|string> $numbers
+     * @param int|string|array<int, int|string> ...$numbers
      */
     public function encode(...$numbers): string;
 
-    /** Decode a hash to the original parameter values. */
+    /**
+     * Decode a hash to the original parameter values.
+     * @return array<int, int>|array{}
```

Comment:
This `|array{}` is totally useless. An empty array satisfies `array<int, int>`

Reply:
d11efb3565433509d1c76c0277c48807e116992c

Reply:
Decode method can return [string](https://github.com/vinkla/hashids/blob/d11efb3565433509d1c76c0277c48807e116992c/src/Hashids.php#L176). Thus return type should be `int[]|string[]` or `array<int|string>`.

Reply:
that would be `array<int|string>` because `int[]|string[]` does not allow mixing strings and integers in the same array.

Reply:
Is it good to mix int and strings? Maybe only strings is fine?